packages/platforms/legacy-adapter-player/src/android-legacy-api/api.ts

```diff
@@ -0,0 +1,71 @@
+import { Msg } from './msg';
+
+import { inbound } from './inbound';
+import { outbound } from './outbound';
+
+interface PlayerData {
+  player_version: string;
+  model: string;
+  platform: string;
+  uuid: string;
+  android_version: string;
+}
+
+import { Logger } from '../utils';
+
+const logger = new Logger('Api');
+
+const msg = new Msg(window.parent, inbound, outbound);
+
+export async function getPlayerData(): Promise<PlayerData> {
+  logger.log('getPlayerData');
```

Comment:
I guess these logs will be removed later as they could create memory issues in the device.

Reply:
The logger is disabled by default and activated per build-time env variable so no mem leaks (caused by this at least 😄 ) in production.

Reply:
In dev mode the logs are valuable since the debugging experience on device for the legacy android player is very poor.